From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dominik Csapak <d.csapak@proxmox.com>
Date: Thu, 2 Apr 2020 07:10:18 +0000
Subject: [PATCH] adapt webpack config

adds 'cli-engine' to build (we use it in our wrapper)
and target 'node' since we will use it on the cli

Signed-off-by: Dominik Csapak <d.csapak@proxmox.com>
---
 webpack.config.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/webpack.config.js b/webpack.config.js
index a22c99b..9209159 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -4,8 +4,9 @@ const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 /** @type {import("webpack").Configuration} */
 module.exports = {
     mode: "none",
+    target: "node",
     entry: {
-        eslint: ["core-js/stable", "regenerator-runtime/runtime", "./lib/linter/linter.js"]
+        eslint: ["core-js/stable", "regenerator-runtime/runtime", "./lib/linter/linter.js", "./lib/cli-engine/index.js"]
     },
     output: {
         filename: "[name].js",
-- 
2.20.1
